src/Test/Runner.elm

```diff
@@ -374,9 +373,9 @@ type Shrinkable a
 {-| Given a fuzzer, return a random generator to produce a value and a
 Shrinkable. The value is what a fuzz test would have received as input.
 -}
-fuzz : Fuzzer a -> Random.Pcg.Generator ( a, Shrinkable a )
```

Comment:
Good point, I hadn't realised this was exported. I'll change that back to the original signature.